openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -496,3 +544,165 @@ class CourseOverviewTab(models.Model):
     """
     tab_id = models.CharField(max_length=50)
     course_overview = models.ForeignKey(CourseOverview, db_index=True, related_name="tabs")
+
+
+class CourseOverviewImageSet(TimeStampedModel):
+    """
+    Model for Course overview images. Each column is an image type/size.
+
+    You should basically never use this class directly. Read from
+    CourseOverview.image_urls instead.
+
+    Special Notes on Deployment/Rollback/Changes:
+
+    1. By default, this functionality is disabled. To turn it on, you have to
```

Comment:
Mostly because it's the first time out, and I wanted to run it a little while before making it the default for everyone. I figured that a follow-on PR could do a data migration to create a default config if none already existed.